jbpm-persistence/jbpm-persistence-api/src/main/java/org/jbpm/persistence/api/integration/InstanceView.java

```diff
@@ -39,4 +39,6 @@
      * PersistentEventManager will invoke it before delivering if it was not done before.
      */
     void copyFromSource();
+    
+    String getCompositeId();
```

Comment:
Ok to the javadoc
Since @override is not the purpose of that JIRA, Im only adding to the classes that needs to be changed anyway, but not adding a new class to the modified set just because adding override.